src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -37,12 +38,18 @@ QtObject:
     result.metadata = metadata
     result.setup()
 
-  proc newTransactionActivityEntry*(tr: ref Item, metadata: backend.ActivityEntry): ActivityEntry =
+  proc newTransactionActivityEntry*(tr: ref Item, metadata: backend.ActivityEntry, fromAddresses: seq[string]): ActivityEntry =
     new(result, delete)
     result.multi_transaction = nil
     result.transaction = tr
     result.isPending = metadata.payloadType == backend.PayloadType.PendingTransaction
     result.metadata = metadata
+    result.activityType = backend.ActivityType.Send
```

Comment:
@stefandunca isn't this supposed to come from the backend?

Reply:
Yes but didn't want to current work to be blocked by the activity metadata plan, this TODO covers it: https://github.com/status-im/status-desktop/blob/951fa8b0f4d145c5825fa35bdd406360498b739b/src/backend/activity.nim#L147

In the backend, it is already set. I was planning in my current PR to fix this, will replace it when I make the change or @Cuteivist, if you want you can fix it yourself.

Reply:
@Cuteivist, thanks for the heads up, I will take a look later on